packages/react/src/publication/useWhoCollectedPublication.ts

```diff
@@ -0,0 +1,27 @@
+import { useWhoCollectedPublicationQuery, WalletFragment } from '@lens-protocol/api-bindings';
+
+import { PaginatedReadResult, PaginatedArgs, usePaginatedReadResult } from '../helpers';
+import { useSharedDependencies } from '../shared';
+import { DEFAULT_PAGINATED_QUERY_LIMIT } from '../utils';
+
+type UseWhoCollectedPublicationArgs = PaginatedArgs<{
+  publicationId: string;
+  observerId?: string;
+}>;
+
+export function useWhoCollectedPublication(
+  args: UseWhoCollectedPublicationArgs,
+): PaginatedReadResult<WalletFragment[]> {
+  const { apolloClient } = useSharedDependencies();
+
+  return usePaginatedReadResult(
+    useWhoCollectedPublicationQuery({
+      variables: {
+        publicationId: args.publicationId,
+        observerId: args?.observerId,
```

Comment:
nit: elvis operator not require as `args` is always an `object` (same on the live below)